infra/templates/az-isolated-service-single-region/variables.tf

```diff
@@ -0,0 +1,140 @@
+// ---- General Configuration ----
+
+variable "admin_subscription_id" {
+  description = "The ID of the subscription in which to deploy admin resources"
+  type        = string
+}
+
+variable "resource_group_location" {
+  description = "The deployment location of resource group container all the resources"
+  type        = string
+}
+
+variable "name" {
+  description = "The name of the deployment. This will be used across the resource created in this solution"
+  type        = string
+}
+
+
+
+// ---- App Service Environment Configuration ----
+
+variable "app_service_environment_name" {
+  description = "The name of the App Service Environment in which to deploy the app services"
+  type        = string
+}
+
+variable "app_service_environment_resource_group" {
+  description = "The resource group of the App Service Environment in which to deploy the app services"
+  type        = string
+}
+
+
+
+// ---- App Service Configuration ----
+
+variable "app_service_name" {
+  description = "The name key value pair where the key is the name assigned to the app service and value is the source container"
+  type        = map(string)
+}
+
+
+// ---- Service Plan Configuration ----
+
+variable "scaling_values" {
```

Comment:
We should rename this to `monitoring_dimension_values` so the naming convention is more accurate to it's use.